src/drt/src/dr/FlexDR.cpp

```diff
@@ -1874,14 +1874,22 @@ void FlexDR::end(bool done)
   const ULL totMCut = std::accumulate(mCut.begin(), mCut.end(), ULL(0));
 
   if (done) {
-    logger_->metric("drt::wire length::total",
-                    totWlen / getDesign()->getTopBlock()->getDBUPerUU());
-    logger_->metric("drt::vias::total", totSCut + totMCut);
+    logger_->metric("route__drc_errors", getDesign()->getTopBlock()->getNumMarkers());
+    logger_->metric("route__wirelength", totWlen / getDesign()->getTopBlock()->getDBUPerUU());
+    logger_->metric("route__vias", totSCut + totMCut);
+    logger_->metric("route__vias__singlecut", totSCut);
+    logger_->metric("route__vias__multicut", totMCut);
   }
+  else if (VERBOSE > 0) {
```

Comment:
I don't think we should make the metrics depend on VERBOSE which controls user output.

Reply:
Initially I thought we could use the VERBOSITY to control metrics verbosity too. However I agree it should not depend on the variable.  Moreover, the intermediate reports are good to have in the report. We will introduce a separate verbosity control for metrics as a separate PR.